aziot/test-files/init/dps-symmetric-key/device-id

```diff
@@ -0,0 +1 @@
+aziot-identity-service|aziot-identity-service|aziot-identity-ser
```

Comment:
nit: missing newline

Reply:
looks like this applies to all `device-id`'s across the various tests

Reply:
It's not supposed to have a newline. It's a binary file. The test data just happens to look like an ASCII string.

Reply:
oh, I totally wouldn't have guessed that. okie all good then, carry on